config/uiowa.edu/user.role.webmaster.yml

```diff
@@ -124,7 +125,6 @@ permissions:
   - 'delete any person content'
   - 'delete any remote_video media'
   - 'delete any twitter media'
-  - 'delete any video media'
```

Comment:
Can you clarify why this was changed?

Reply:
looks like it got pushed down to line 141 due to all of the uiowa_* block content permissions

Reply:
Ah. Got it.